dotcom-rendering/src/components/AdSlot.tsx

```diff
@@ -4,6 +4,7 @@ import type { SlotName } from '@guardian/commercial';
 import { ArticleDisplay } from '@guardian/libs';
```

Comment:
I noticed that we mostly define css above the component, but sometimes inside the component. I've moved the css definitions to above the component for all ad slots for consistency